sleap/gui/learning/dialog.py

```
@@ -637,6 +637,20 @@ def get_items_for_inference(self, pipeline_form_data) -> runners.ItemsForInferen
             )
         return items_for_inference
 
+    def _validate_id_model(self) -> bool:
+        """Make sure we have instances with tracks set for ID models."""
+        if not self.labels.tracks:
+            message = "Cannot run ID model training without tracks."
```

Ruff (0.8.2)

Local variable `message` is assigned to but never used

Remove assignment to unused variable `message`

(F841)